monai/apps/auto3dseg/algorithms/templates/dints/configs/hyper_parameters.yaml

```diff
@@ -0,0 +1,61 @@
+---
+# hyper-parameters
+amp: true
+bundle_root: null
+ckpt_path: "$@bundle_root + '/model_fold' + str(@fold)"
+data_file_base_dir: null
+data_list_file_path: null
+determ: false
+fold: 0
+input_channels: null
+learning_rate: 0.025
+num_images_per_batch: 2
+num_iterations: 40000
+num_iterations_per_validation: 500
+num_patches_per_image: 1
+num_sw_batch_size: 2
+output_classes: null
```

Comment:
output_classes was used as output_channels/out_channels in the train.py

Reply:
sorry I did not find the corresponding line of output_channels/out_channels. can you point it out? thanks!